### DISCUSSION TITLE
"View score on the web" option?

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#30029] [Q&A]
Basically i like using different websites to check how much pp my score would have with different calculations, just for fun yk?
This would also help with just overall locating scores on the web since all you would have to do is click a button in game to find the score on the web.

This would be availble either on the footer of the score result screen, or near the beatmap name (tho the first is better since its pointless to add things to the current SRS since it will get replaced soon)

**Comment by peppy** [comment]
Definitely need this in multiple places. We already have pretty good coverage for opening beatmaps externally but not yet for scores.
